TestApi/TestApi.IntegrationTests/appsettings.json

```diff
@@ -6,8 +6,20 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "Services": {
+    "TestApiUrl": "https://TestApi_AC/"
+  },
+  "ZapConfiguration": {
+    "ApiAddress": "127.0.0.1",
+    "ApiPort": "8080",
+    "ApiConfigPath": "../../../../TestApi.IntegrationTests/zapapi-config.xml",
+    "ServiceName": "TestApi_AC",
+    "SolutionFolderName": "TestApi",
+    "SetUpProxy": false,
+    "ActiveScan": false,
+    "ZapScan": false
+  },
   "AllowedHosts": "*",
   "AzureAD:Authority": "https://login.microsoftonline.com/",
-  "IsLive": false,
-  "ConnectionStrings:TestApi": "Server=(LocalDb)\\MSSQLLocalDB;Database=TestApi;Trusted_Connection=True;"
```

Comment:
The connection string is needed to run locally

Reply:
This settings does not override appsettings in TestApi solution, it works without setting this so felt redundant and removed it